types/meteor/react-meteor-data.d.ts

```diff
@@ -11,3 +11,17 @@ export function withTracker<TDataProps, TOwnProps>(
  * Requires react-meteor-data 2.0.0 or later
  */
 export function useTracker<TDataProps>(getMeteorData: () => TDataProps, deps?: React.DependencyList): TDataProps;
+
+/**
+ * Requires react-meteor-data 2.4.0 or later
+ */
+export function useSubscribe(name?: string, ...args: any[]): () => boolean;
```

Comment:
even here we could probably change `any` to `unknown`. I think making the interface as modest as possible is a good approach.

Reply:
It actually matches the [`Meteor.subscribe`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/837dfb86fcd988648e97d2eea2a522565da23954/types/meteor/meteor.d.ts#L372-L384) type, so I'd say it's fine.